### ISSUE TITLE
Pknx doesn't save Pokémon edits 

### ISSUE THREAD

**Issue by Mrgamer2240** [kwsch/pkNX#377]
I tried editing Pokémon types and was successful once but never again after that, I closed Pknx before copying the edited files to the SD card.

**Comment by kwsch** [comment]
Are the changes still there in the pkNX user interface if you reopen it? Be sure you hit save rather than exit out of the window.

**Comment by Mrgamer2240** [comment]
Yep changes are there double checked, but the type nor stat changes appear in game I saved my edited files to my game file then put it on my SD card again,  I saved this time but it's still inactive in game I checked my Pknx Pokémon option and it's all there but it never works in game.

**Comment by kwsch** [comment]
If pkNX remembers the changes, then the issue isn't with the program, it's with the way you're trying to get your console to use the edited files. If it is not using them, well, double check how Atmosphere needs layeredfs set up.

**Comment by Mrgamer2240** [comment]
Everything is set up correctly,  but maybe not the file placement of the edited files I put those in this order, Atmosphere/Contents/01008DB008C2C000/RomFS/bin/pml/evolution. Is there an error I did somewhere? This was working perfectly fine a few weeks ago.

**Comment by kwsch** [comment]
pkNX will output the edited files to the title ID folder for simple copy/paste.

Pretty sure you have it right, `atmosphere/contents/`. Try with other files as well.
Not sure what edits you did, they might not even "work" or be complete?

**Comment by Mrgamer2240** [comment]
The edits were stats, types Evo, learnsets, etc. I know not of any other method for putting file on my switch other than using "USB File Transfer" or taking out the SD card and inserting it into the pc. The only file this works for is evo_818/Inteleon.

**Comment by m4tata** [comment]
I also have this issue for me it seems like the evolutions file or the typings file doesnt save, i also have edited evolutions, typings and learnsets but the only thing that got changed in game was the learnsets. when i go into noctowl (the one i edited) everything is there the changed ice flying typing etc. if i look in the saved folder all i can see is:
romfs\bin\archive\field\resident/ data_table.gfpak
and
romfs\bin\pml\waza_oboe/waza_oboe_total.bin
and
romfs\bin\pml\personal/personal_total.bin

**Comment by kwsch** [comment]
Relative paths are not helpful; check the output directory (titleID/...).

**Comment by Lusamine** [comment]
This looks to be a similar complaint as https://github.com/kwsch/pkNX/issues/386. I just tried editing SWSH Pokemon on latest commit of pkNX and it is working now, so I would suggest seeing that issue for my suggestions on how to get this to work.
